src/test/safe.dom.upgrade.test.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+import Web3 from 'web3'
+import GnosisSafeSol from '@gnosis.pm/safe-contracts/build/contracts/GnosisSafe.json'
+import {
+  DEFAULT_FALLBACK_HANDLER_ADDRESS,
+  getEncodedMultiSendCallData,
+  SAFE_MASTER_COPY_ADDRESS,
+} from '~/logic/contracts/safeContracts'
+import { ZERO_ADDRESS } from '~/logic/wallets/ethAddresses'
+
+
+describe('DOM > Feature > Upgrade a Safe', () => {
```

Comment:
I'm not sure about the `'DOM'` part of the description. It's a feature, but not related to the components or the DOM.

Reply:
Yeah the dom part is a little confusing

Reply:
done